cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -0,0 +1,128 @@
+from marshmallow import fields, post_load
+
+from pcluster.config.imagebuilder_config import (
+    ImageBuilderBucketLogConfig,
+    ImageBuilderBuildConfig,
+    ImageBuilderChefCookbookConfig,
+    ImageBuilderComponentsConfig,
```

Comment:
I see there is the ImageBuilder prefix in all the components
I think you can avoid it and have something like:
```
BucketLogConfig,
BuildConfig,
...
ImageBuilderConfig
```

Reply:
Done